avaframe/out3Plot/outCom3Plots.py

```diff
@@ -146,15 +146,15 @@ def generateCom1DFAPathPlot(avalancheDir, cfgPath, avaProfileMass, dem, paraboli
              label='_bottom extension', lw=2, path_effects=[pe.Stroke(linewidth=3, foreground='g'), pe.Normal()])
     ax1.plot(avaProfileMass['x'][indStart:indEnd+1], avaProfileMass['y'][indStart:indEnd+1], '-y.', zorder=20,
              label='_Center of mass path', lw=2, path_effects=[pe.Stroke(linewidth=3, foreground='k'), pe.Normal()])
-    if splitPoint != '':
+    if not splitPoint.get('isTopSplitPoint', False):
```

Comment:
This is really hard to read, since you are negative testing for a negative. Please change to `is`... `True`